### PR TITLE
Feat / a11y improve html structure for VideoListItem

### PR DESCRIPTION
**Pull request by langemike**
This change applies the same HTML with accessibility improvement to the `VideoListItem` component. Taking the `Card` component as the base. You should [see the Card.tsx component](https://github.com/Videodock/ott-web-app/blob/feat/a11y-sprint-2/packages/ui-react/src/components/Card/Card.tsx#L98) for reference.

I also converted the related list to a list (`ul` > `li`).

While working on it I also noticed that both the `VideoListItem` and `Card` component are almost the same. More than I originally anticipated.

**Differences**
- Technically: Loading state is missing
- Technically: Inconsistent prop names (isActive/isCurrent & isCurrentLabel/isActiveLabel)
- Visually: `VideoListItem` is vertically orientated and `Card` horizontally

**Similarities**
- They are (HTML) structurally the same - besides Card having a dynamic h2/h3
- Tags
  - In both components duplicate code exist to determine the tag content
  - The Tag component should ideally be used for this purpose, but it was not the case in the Card component. A regular `<div>` with tag styling is used instead.
  - The Tag styling is a mess and is duplicated across Card.module.scss and VideolistItem.module.scss. 
  - Styles previously applied in `VideolistItem.module.scss` where overruled because of CSS specificity on the Tag component. The Tag component is now not used anymore on the `VideoListItem`, because applying this component in the Card will take me another hour to get the styling identical. When you perform this taks, you will easily go in to a chain reaction, also wanting to solve the other issues described above. I vouch for fixing everything in one go and reserve time for it.
  
Based on this insight I still think we need to combine them eventually and separate the styling.

Ticket: [OTT-545](https://videodock.atlassian.net/browse/OTT-545)

New ticket created for Tag clean-up: https://videodock.atlassian.net/browse/OTT-770

[OTT-545]: https://videodock.atlassian.net/browse/OTT-545?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ui-react/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -95,6 +95,8 @@ function Card({
     }
   };
 
+  const heading = React.createElement(`h${headingLevel}`, { className: classNames(styles.title, { [styles.loading]: loading }) }, title);
```
Comment (langemike) [inline packages/ui-react/src/components/Card/Card.tsx:98]: The dynamic heading level was only applied once previously. So I also added this as a bonus

File: packages/ui-react/src/components/VideoLayout/VideoLayout.tsx [hunk 2]
```diff
@@ -139,7 +139,6 @@ const VideoLayout: React.FC<Props> = ({
     ) : (
       <div className={styles.relatedVideosList}>
         <VideoList
-          className={styles.videoList}
```
Comment (langemike) [inline packages/ui-react/src/components/VideoLayout/VideoLayout.tsx:142]: Unused CSS definition

File: packages/ui-react/src/components/Card/Card.tsx (hunk 1 above)
Comment (royschut) [reply packages/ui-react/src/components/Card/Card.tsx:98]: We could make a `<Heading />` component actually, to be used anywhere. (maybe not now)

**Review by royschut — APPROVED** [review]
Nice Mike!

**Review by ChristiaanScheermeijer — APPROVED** [review]

**Pull request merged**